src/test/scala/edu/ie3/mobsim/model/ElectricVehicleSpec.scala

```diff
@@ -55,11 +51,11 @@ class ElectricVehicleSpec extends UnitSpec with TripSimulationTestData {
               ) =>
             simulationStart shouldBe givenSimulationStart
             id shouldBe "test_car"
-            model shouldBe "cool_producer cool_model"
-            batteryCapacity shouldBe givenModel.capacity
-            acChargingPower shouldBe givenModel.acPower
-            dcChargingPower shouldBe givenModel.dcPower
-            consumption shouldBe givenModel.consumption
+            evType.model shouldBe "cool_model"
+            evType.capacity shouldBe givenModel.capacity
+            evType.acPower shouldBe givenModel.acPower
+            evType.dcPower shouldBe givenModel.dcPower
+            evType.consumption shouldBe givenModel.consumption
             homePoi shouldBe givenHomePoi
```

Comment:
This can now be abbreviated by:

```suggestion
            evType shouldBe givenModel
```